srcpkgs/mabel/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'mabel'
+pkgname=mabel
+version=0.1.3
+revision=1
+build_style=go
+go_import_path="github.com/smmr-software/${pkgname}"
```

Comment:
```suggestion
go_import_path="github.com/smmr-software/mabel"
```

The package name is not variable, and the variable is longer than the actual name.